content/meetup-december-2016.md

```diff
@@ -0,0 +1,68 @@
+Title: Meet Up Summary (December, 2016)- Advice to the young'uns
```

Comment:
I'd recommend keeping the title standard and instead using the summary for going off script.